contrib/hugin.lua

```diff
@@ -0,0 +1,66 @@
+--[[
+  Hugin storage for darktable 
+
+  copyright (c) 2014  Wolfgang Goetz
+  copyright (c) 2015  Christian Kanzian
+  
+  darktable is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+  
+  darktable is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+  
+  You should have received a copy of the GNU General Public License
+  along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+
+--[[
+HUGIN
+Add a new storage option to send images to hugin. 
+Images are exported to darktable tmp dir first. 
+
+USAGE
+* require this file from your main luarc config file.
+
+This plugin will add a new storage option and calls hugin after export.
+]]
+
+dt = require "darktable"
+
+-- should work with lua version 2.0.0

```

Comment:
it's not the lua version that we check, it's the DT API version...

please correct/remove the comment i'm afraid it might confuse other readers